src/App.css

```diff
@@ -1,5 +1,10 @@
 .App {
   text-align: center;
+  display: flex;
+  flex-direction: column-reverse;
```

Comment:
Flex! Great choice. I love flex! 

I see that you used `flex-direction: column-reverse`. What kinds of implications would using `column-reverse` vs `column` having moving forward?